File: Level1/2016년.java
class Solution {
    String[] weekday = {"SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"};
    int[] dayOfMonth = {31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
    public String solution(int a, int b) {
        int day = b;
        for (int i = 0; i < a - 1; i++) {
            day += dayOfMonth[i];
        }
        String answer = weekday[(day + 4) % 7];
        return answer;
    }
}

File: Level1/2016년.py
weekday = ['SUN', 'MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT']
day_of_month = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]

def solution(a, b):
    day = b
    for i in range(0, a - 1):
        day += day_of_month[i]
    answer = weekday[(day + 4)% 7]
    return answer
